search/src/Components/auth/auth_prompt.js

```diff
@@ -49,11 +49,11 @@ export default function AuthPrompt(props) {
     <div>
       <div style={{display: 'flex', flexDirection: 'row', alignItems: 'center'}}>
         Please&nbsp;
-        <Button variant="outlined" color="primary" onClick={handleLogin}>
+        <Button id="sign_in_button" variant="outlined" color="primary" onClick={handleLogin}>
```

Comment:
Yeah, so this is definitely the kind of thing where the preferred testing code should _not_ use the ID. Rather, you should use use `cy.findByRole('button', {name: /Sign In/})`.

See https://testing-library.com/docs/queries/byrole/ which mentions the default `role` of buttons and the "accessible name" being the text of the button. See also "Rules of ARIA attribute usage by HTML element" at [the aria spec](https://www.w3.org/TR/html-aria).

Reply:
It worths trying out for different combination to actually use the api
![Screen Shot 2021-05-07 at 1 06 43 pm](https://user-images.githubusercontent.com/51065213/117392280-14ade400-af35-11eb-929b-2cb02c173f4c.png)

Reply:
![Screen Shot 2021-05-07 at 1 10 58 pm](https://user-images.githubusercontent.com/51065213/117392566-ad446400-af35-11eb-8350-742cbbee18a3.png)

Reply:
Strange that there are multiple elements but we can't see them listed...?